lib/ably/realtime/connection.rb

```diff
@@ -615,6 +615,8 @@ def connection_resumable?
       def connection_state_available?
         return true if connected?
 
+        return false if time_since_connection_confirmed_alive? > connection_state_ttl
```

Comment:
Shouldn't this check include also the idle interval? Much like JS does https://github.com/ably/ably-js/blob/c55b4518d6f096e4bde90afcd04c5ba35d466f3c/common/lib/transport/connectionmanager.js#L706

Reply:
Oops, yes, I forgot the idle interval. Thanks, good spot, I was being a bit too strict there.